src/middlewared/middlewared/plugins/service_monitor.py

```diff
@@ -137,40 +147,58 @@ def run(self):
             enabled = self.isEnabled(self.name)
 
             self.logger.debug("[ServiceMonitorThread] connected=%s started=%s enabled=%s", connected, started, enabled)
```

Comment:
This is like spamming logs every `self.frequency`, can we log only if something has changed or gone bad? Or change it to `logger.trace`?

Reply:
Was there originally, can use `trace`.